tests/test_sprites.py

```diff
@@ -325,3 +325,14 @@ def test_rotatable_base_sprite():
 
     test_sprite.rotate(1)
     assert test_sprite.rotation == 1
+
+
+def test_sprite_in_main():
```

Comment:
This doesn't fail if you revert the change. Doing some experimentation to find a way to force it to fail.